t01_final_story.py

```diff
@@ -552,10 +552,6 @@
     else:
         print("Unlucky, you're dead. Very sad")
         quit()
-
-# TODO Make sure to add the additional check if the user makes the "bad" choice!
-
-# TODO Don't forget to check if your user is dead at the end of your chapter!
```

Comment:
You should not have edited the t01_final_story.py file. Please revert this code!